CMakeLists.txt

```diff
@@ -168,6 +168,14 @@ include( CMakePushCheckState )
 include( GNUInstallDirs )
 include( TestBigEndian )
 
+cmake_dependent_option(
+   ${_OPT}has_sentry_reporting 
+   "Has a builtin support for sending errors to Sentry"
```

Comment:
Suggest consistency of wording with the option before it:

"Build support for sending errors to Sentry"